include/course.h

```diff
@@ -108,9 +108,9 @@ extern f32 gCourseFarPersp;
 extern f32 gCourseNearPersp;
 #define gCourseNearPersp gCourseNearPersp
 #define GET_COURSE_D_0D0096B8(cc) *(f32*) segmented_to_virtual_dupe_2(&D_0D0096B8[gCurrentCourseId][cc])
-#define GET_COURSE_D_0D009808(cc) *(f32*) segmented_to_virtual_dupe_2(&D_0D009808[gCurrentCourseId][cc])
-#define GET_COURSE_D_0D009418(cc) *(f32*) segmented_to_virtual_dupe_2(&D_0D009418[gCurrentCourseId][cc])
-#define GET_COURSE_D_0D009568(cc) *(f32*) segmented_to_virtual_dupe_2(&D_0D009568[gCurrentCourseId][cc])
+#define GET_COURSE_gCpuOffTrackTargetSpeed(cc) *(f32*) segmented_to_virtual_dupe_2(&gCpuOffTrackTargetSpeed[gCurrentCourseId][cc])
```

Comment:
gCPUOffTrack

Reply:
it's a target speed for when the cpu are gCPUOffTrack so I prefer gCpuOffTrackTargetSpeed

Reply:
I think I actually was just trying to say capitalize cpu as CPU instead of Cpu